src/main/java/org/sagebionetworks/web/server/servlet/SynapseClientImpl.java

```diff
@@ -274,6 +275,22 @@ public EntityBundle getEntityBundleForVersion(String entityId,
 			throw ExceptionUtil.convertSynapseException(e);
 		}
 	}
+	
+	@Override
+	public EntityBundlePlus getEntityBundlePlusForVersion(String entityId,
+			Long versionNumber, int partsMask) throws RestServiceException {
+		try {
+			org.sagebionetworks.client.SynapseClient synapseClient = createSynapseClient();
+			EntityBundle eb = synapseClient.getEntityBundle(entityId, versionNumber, partsMask);
```

Comment:
Verified that synapseClient.getEntityBundle(entityId, versionNumber, partsMask) has correct behavior when versionNumber is null?